i18n/en.json

```diff
@@ -4815,6 +4815,14 @@
     "id": "app.cloud.upgrade_plan_bot_message_single",
     "translation": "{{.UsersNum}} member of the {{.WorkspaceName}} workspace has requested a workspace upgrade for: "
   },
+  {
+    "id": "app.collection.add_collection.exists.app_error",
+    "translation": "Only allowed single collection type per plugin."
```

Comment:
Should we relax this to support an arbitrary number of collections per plugin, just like we support multiple topics per collection?